module/Application/src/Application/Service/UserUpdateService.php

```diff
@@ -19,7 +19,10 @@ public function __construct(EntityManager $entityManager, SecurityLogger $securi
         $this->securityLogger = $securityLogger;
     }
 
-    public function updateUser($userId, $newValues)
+    /**
+     * @param array<mixed> $newValues
+     */
+    public function updateUser(int $userId, array $newValues)
```

Comment:
I'd move these both to comment-based typehints. They come from the controller so we're not in control of their types so, for example, `$userId` might come through as a string. If it does, that isn't really a problem because everything we use it for could coerce it. But by adding the proper PHP typing, PHP would throw an error in this instance.

By using comments you can improve the typing without making any real code changes